conftest.py

```diff
@@ -42,17 +46,14 @@ def profile_copy(driver, pref_set):
     path = driver.capabilities['moz:profile']
     try:
         shutil.rmtree(path_profile_dest)
-        time.sleep(5)
+        time.sleep(1)
     except:
         pass
-    # TODO: remove this
-    time.sleep(30)
     print('\n====================================')
```

Comment:
we can eventually drop this as well.  it was just there to verify our path was correcto